app/components/newsletter/newsletter-header-overview.js

```diff
@@ -181,6 +189,34 @@ export default class NewsletterHeaderOverviewComponent extends Component {
     this.showConfirmPublishAll = false;
   }
 
+  async canSendMailCampaign() {
+    const documentPublicationActivity = await this.args.meeting.internalDocumentPublicationActivity;
+
+    let hasDocumentPublicationDate = isPresent(documentPublicationActivity.plannedDate);
+    let hasNotas = false;
+    let hasThemes = false;
+
+    const agendaitems = await this.args.agenda.agendaitems;
```

Comment:
This is a wonky relation (easily stale) + this will only load up to 20 agendaitems due to default page size limits.
We typically never use it directly.

We could use the same query as used in the child route:
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/routes/newsletter/print.js#L27

Reply:
That query is already setup to filter out announcements when it's a draft so you would only need to check if 1 theme exists.